ocfweb/reservations/mail_templates/officer_notification.jinja

```diff
@@ -0,0 +1,9 @@
+Hello all,
+
+The OCF has received a lab reservation request!
+
+The request was issued by {{request.real_name}} ({{request.contact_email}}) on behalf of the student group {{request.student_group}}.
+They wish to use our lab from {{request.starttime}} until {{request.endtime}} on {{request.date}}.
+Their stated reason is: {{request.reason}}
+
+Would a staffer kindly create an RT ticket and place it in the bod queue so that a vote can be held at the next meeting?
```

Comment:
since these are plain-text email templates, please wrap lines to ~72 chars

Reply:
Fixed